org.freedesktop.Sdk.Extension.php73.json

```diff
@@ -19,12 +35,67 @@
 		"no-debuginfo": true,
 		"strip": true,
 		"prefix": "/usr/lib/sdk/php73",
-		"append-pkg-config-path": "/usr/lib/sdk/php73/lib/pkgconfig"
+		"prepend-path": "/usr/lib/sdk/php73/bin",
+		"prepend-ld-library-path": "/usr/lib/sdk/php73/lib",
+		"append-pkg-config-path": "/usr/lib/sdk/php73/lib/pkgconfig",
+		"env": {
+			"PHP_INI_DIR": "/usr/lib/sdk/php73/etc/php"
+		}
 	},
 	"modules": [
+		{
+			"name": "libldap",
+			"buildsystem": "autotools",
+			"config-opts": [
+				"--disable-slapd",
+				"--disable-static"
+			],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://www.openldap.org/software/download/OpenLDAP/openldap-release/openldap-2.4.48.tgz",
+					"sha256": "d9523ffcab5cd14b709fcf3cb4d04e8bc76bb8970113255f372bc74954c6074d"
+				},
+				{
+					"type": "patch",
+					"path": "openldap-aarch64.patch",
+					"only-arches": [
+						"aarch64"
+					]
+				}
+			],
+			"post-install": [
+				"install -Dm644 LICENSE ${FLATPAK_DEST}/share/licenses/libldap/LICENSE"
+			]
+		},
+		{
+			"name": "libpq",
+			"buildsystem": "simple",
+			"build-commands": [
+				"./configure --prefix=${FLATPAK_DEST} --disable-rpath --with-openssl --without-readline",
+				"cd src/interfaces/libpq",
+				"make -j$FLATPAK_BUILDER_N_JOBS",
+				"make install"
+			],
+			"sources":[
```

Comment:
Lack of space here before the `[`

Just a nit pick, but you have done a space before for the other ones.

Reply:
Yeah but existing code doesn't use a space after `:`.

Maybe we should standardize all of this in another PR ? This one is already huge.

Reply:
Seems good with me.